Objects/bytearrayobject.c

```diff
@@ -321,6 +321,7 @@ bytearray_repeat(PyByteArrayObject *self, Py_ssize_t count)
     PyByteArrayObject *result;
     Py_ssize_t mysize;
     Py_ssize_t size;
+    char *buf;
```

Comment:
```suggestion
    const char *buf;
```
We age not going to mutate it.